homeassistant/components/zha/core/device.py

```diff
@@ -160,6 +165,19 @@ def is_mains_powered(self):
         """Return true if device is mains powered."""
         return self._zigpy_device.node_desc.is_mains_powered
 
+    @property
+    def device_type(self):
+        """Return the logical device type for the device."""
+        return (
```

Comment:
There's logical type enum on zigpy device node descriptor https://github.com/zigpy/zigpy/blob/01a5a66c3df2a502c150def7b8c0ef6399842e0e/zigpy/zdo/types.py#L83

we could do 
```
device_type = self._zigpy_device.node_desc.logical_type
return device_type.name if device_type or "Unknown"
```
The only downside, no space in `EndDevice` name.

Reply:
Yeah I’ll change to that

Reply:
updated